### PR TITLE
Work-about sections

### PR DESCRIPTION
**Pull request by daudi13**
#in this project I:

-Added the work section to .html
-Added about section to .html
-Added each file on a unique commit
-Fixed linter errors
-used flexbox and grid in placing elements
-updated .css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,7 +40,139 @@ <h2 class="headline-section__sub-title">
...
+						<div class="tech__type">html</div>
+						<div class="tech__type">css</div>
+						<div class="tech__type">javaScript</div>
+					</div>
+					<button class="btn see-project">See Project</button>
+				</div>
+			</div>
+			<div class="work-section__card">
+				<div class="work-section__card--img"><img src="/images/SnapshootPortfolio(1).png" alt=""></div>
+				<div class="work-section__card--info">
```
Comment (ReshyResh) [inline index.html:72]: - [x] You did a good job using `flex` to display the elements inside the work cards, however, as per requirement, the `<section>` tag including all the cards should use a `display:grid` property.

File: index.html [hunk 2]
```diff
@@ -36,7 +40,139 @@ <h2 class="headline-section__sub-title">
...
+						<div class="range__circle"></div>
+						<p class="range__txt">2015</p>
+					</div>
+					<p class="general__desc">
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<div class="tech">
+						<div class="tech__type">html</div>
+						<div class="tech__type">css</div>
+						<div class="tech__type">javaScript</div>
```
Comment (ReshyResh) [inline index.html]: - [x] As per requirement, list items should be inside a `<ul>` tag, with each element being a list item `<li>`. Please make sure to apply the changes to all the cards.

File: index.html [hunk 3]
```diff
@@ -36,7 +40,139 @@ <h2 class="headline-section__sub-title">
...
+                <li class="headline-section__social--item"><a href="#" class="social-icon linkedin"><img src="images/medium.svg" alt="medium icon"></a></li>
+                <li class="headline-section__social--item"><a href="#" class="social-icon github"><img src="images/github.svg" alt="github icon"></a></li>
+                <li class="headline-section__social--item"><a href="#" class="social-icon chill"><img src="images/chill.svg" alt="chill icon"></a></li>
+            </ul>
+
+			<button class="btn resume">Get my resume</button>
+        </div>
+		<div class="language-section">
+			<div class="language-section__title"><h4>Language</h4><button class="down"><span class="glyphicon glyphicon-chevron-down"></span></button></div>
+			<div class="language-section__box-sec">
```
Comment (ReshyResh) [inline index.html]: - [x] The section under the `Get my resume` button should be a list, with the elements containing items being another list.
The final structure should be as follows:
  - ul
     - li (Language)
         - ul
             - li (Javascript)
             - li (HTML)
             - li (CSS)
         - /ul
      - /li
      -  ...

File: style.css [hunk 4]
```diff
@@ -124,3 +131,169 @@ main {
...
+}
+
+.work-section__card {
+  padding: 16px 16px 28px 16px;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
```
Comment (ReshyResh) [inline style.css:162]: - [x] Please make sure to follow the desing guidelines and align the pictures in the middle, currently they're leaning on the left side.
- [x] Also please make sure to add some margin between each individual card, to space it from the one right under it.

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @daudi13  , 

You did a really good good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

To highlight:
  - ✔Descriptive commit messages.
  - ✔Design matches template requirements.
  - ✔Used `flex` correctly to display the content of the cards.


## Required Changes ♻️
_Please check the comments under the review._

----

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mateo951) [reply index.html:72]: Addressed ✔

File: style.css (hunk 4 above)
Comment (mateo951) [reply style.css:162]: Addressed ✔

File: index.html (hunk 3 above)
Comment (mateo951) [reply index.html]: Addressed ✔

File: index.html (hunk 2 above)
Comment (mateo951) [reply index.html]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
Hi @daudi13, 

Awesome job at making the required changes asked by the previous reviewer 🧐✔Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**